dotcom-rendering/src/lib/decidePalette.ts

```diff
@@ -1444,6 +1444,7 @@ const textCarouselTitle = (format: ArticleFormat): string => {
 	return pillarPalette[format.theme].main;
 };
 
+/** @deprecated this has been moved to the theme palette (--drop-cap) */
```

Comment:
You can also remove it completely if you want like [here](https://github.com/guardian/dotcom-rendering/pull/9633/files) but up to you really!